src/ch/imedias/rsccfx/model/Rscc.java

```diff
@@ -33,13 +33,19 @@
   private StringProperty key = new SimpleStringProperty();
   private String keyServerIp;
   private String keyServerHttpPort;
+  private String pathToStunDumpFile ;
 
   /**
    * Initializes the Rscc model class.
    */
   public Rscc(SystemCommander systemCommander) {
     this.systemCommander = systemCommander;
     defineResourcePath();
+
+    keyServerSetup("localhost", "800");
+
+    requestTokenFromServer();
```

Comment:
maybe it isn't the best idea to call requestTokenFromServer already in the constructor of the model, else it will setup a server and if two people want to connect to each other, they will instead connect to themselves since both are running the server (loop to their own machine). this call should be when the scene is switched to the requestview and it should also be made sure that killconnection is called once the requestview is left